app/controllers/courses_controller.rb

```diff
@@ -41,6 +41,13 @@ def duplicate
     redirect_to @course.copy, flash: { success: t('courses.copied_succesfully') }
   end
 
+  def destroy
+    @course = Course.find(params[:id])
```

Comment:
the course will be automatically loaded by a library called `cancancan`, which is activated on line 4. Try if it still works after deleting this line.

Reply:
I tried, it works nice.